src/lighteval/models/model_config.py

```diff
@@ -219,6 +219,25 @@ def init_configs(self, env_config: EnvConfig):
         return self._init_configs(self.base_model, env_config)
 
 
+@dataclass
+class VLLMModelConfig:
+    pretrained: str
+    gpu_memor_utilisation: float = 0.9
+    batch_size: int = 1
```

Comment:
We use -1 in the other configs to force it to being set somewhere, would be good to be homogeneous

Reply:
Same for all other params